### ISSUE TITLE
Many NullPointer AppCrashes

### ISSUE THREAD

**Issue by HansKrywaa** [randdusing/cordova-plugin-bluetoothle#615]
We received many many Crashes caused of this Plugin by a NullPointer Exception:

**java.lang.NullPointerException: Attempt to invoke virtual method 'void android.bluetooth.BluetoothGattServer.close()' on a null object reference**

```
com.randdusing.bluetoothle.BluetoothLePlugin$1.onReceive BluetoothLePlugin.java:2798
android.app.LoadedApk$ReceiverDispatcher$Args.lambda$getRunnable$0$LoadedApk$ReceiverDispatcher$Args LoadedApk.java:1646
android.app.-$$Lambda$LoadedApk$ReceiverDispatcher$Args$_BumDX2UKsnxLVrE6UJsJZkotuA.run
android.os.Handler.handleCallback Handler.java:883
android.os.Handler.dispatchMessage Handler.java:100
android.os.Looper.loop Looper.java:237
android.app.ActivityThread.main ActivityThread.java:8019
java.lang.reflect.Method.invoke Method.java
com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run RuntimeInit.java:493
com.android.internal.os.ZygoteInit.main ZygoteInit.java:1100
```

<img width="1347" alt="Bildschirmfoto 2020-06-19 um 12 01 47" src="https://user-images.githubusercontent.com/20899975/85121207-af6dd900-b224-11ea-9acd-bc4d0e467cca.png">

**Comment by randdusing** [comment]
Looks to be related to this change: https://github.com/randdusing/cordova-plugin-bluetoothle/commit/4382eb0a5a2f2c3e5345887f382187d33ea7f722
Looks like TURNING_OFF can be triggered when not expected. A simple null check should suffice.

**Comment by HansKrywaa** [comment]
Is it possible that you can release a quick fix Version for this?

**Comment by randdusing** [comment]
PRs are always welcome, but I may be able to raise one myself this week

**Comment by HansKrywaa** [comment]
Just create the PR. That was not possible for my Java knowledge😃

**Comment by randdusing** [comment]
I've released 4.15.14, so this should be fixed now.

**Comment by HansKrywaa** [comment]
I was able to force the Crash by toggling Airplane Mode. This doesn't crashed now to think it works. Thanks 😊
